src/App/App.Autorest/README.md

```diff
@@ -62,8 +62,10 @@ identity-correction-for-post: true
 resourcegroup-append: true
 nested-object-to-string: true
 auto-switch-view: false
+# Disable transform IdentityType as GET+PUT can not replace patch
+# 1. Organizations_CreateOrUpdate can not update resources
+# 2. the input schemas of PUT and PATCH are different
 disable-transform-identity-type: true
```

Comment:
Can you elaborate why this attribute is not removed?

Reply:
cc @dolauli

Reply:
LGTM